tests/conftest.py

```diff
@@ -39,7 +39,11 @@ def wrapper(*args, **kwargs):
             with pytest.raises(ValueError) as info:
                 func(*args, **kwargs)
             exception_str = str(info.value)
-            exception_str = exception_str.replace(__tests__ + '/', '')
+            if platform.system() == 'Windows':
+                exception_str = exception_str.replace(__tests__ + '\\', '')
+                exception_str = exception_str.replace('\\', '/')
```

Comment:
This is a little bit weird since it means we will have to change all `\` to `/` in test assertions, which is why I preferred customizing the message itself depending on the OS. Not a big deal though, as these test cases change rarely and only one has a `\\` in the exception message now.